lib/ansible/executor/task_executor.py

```diff
@@ -1249,8 +1249,7 @@ def start_connection(play_context, options, task_uuid):
     else:
         try:
             result = json.loads(to_text(stderr, errors='surrogate_then_replace'))
-        except getattr(json.decoder, 'JSONDecodeError', ValueError):
-            # JSONDecodeError only available on Python 3.5+
+        except (json.decoder.JSONDecodeError, ValueError):
```

Comment:
This should probably go in a separate PR, since the only relation it has with Python 3.5 is the comment. The fallback behavior hasn't been needed since ansible-core 2.11 when Python 2.7 was still unofficially supported as a controller.

That said, when it is included, there's a simpler form to use:

```suggestion
        except json.decoder.JSONDecodeError:
```

Reply:
Added back. Good spotting, this was a weird typo :)